### ISSUE TITLE
Fix docker compose wait-for-it.sh script 

### ISSUE THREAD

**Issue by stevesap** [corona-warn-app/cwa-server#339]
### Current status

The [execution ](https://github.com/corona-warn-app/cwa-server/blob/7d1a5a4e293a83a388d126f32a1c229f13467437/docker-compose.yml#L87)of the existing [wait-for-it.sh script](https://github.com/corona-warn-app/cwa-server/blob/master/scripts/wait-for-it/wait-for-it.sh) was reverted in https://github.com/corona-warn-app/cwa-server/pull/338 since it was not executing properly in windows.

Error message from ``docker-compose run create-bucket`` is:

``No such file or directory``

### Expected Behavior

wait-for-it.sh should execute properly in linux, mac and windows environments

**Comment by stevesap** [comment]
@AlexanderBarbie would you know how to fix this perhaps?

**Comment by AlexanderBarbie** [comment]
Well, that's why you don't develop software on Windows systems. 😄 
On Windows you have to use path\to\wait-for-it instead of path/to/wait-for-it (unix), if you mount the volume. I will install a windows vm and try to fix it.

**Comment by phros** [comment]
I got the same running linux

**Comment by AlexanderBarbie** [comment]
@pithumke can you please tell me which Docker version you are using and did you add the wait-for-it folder under Resources > File Sharing in the Docker settings?

And did you get the error 

> No such file or directory/bin/env: bash


This can be a problem with the line ending. Windows changes the line endings of downloaded files and then you can't start the bash script in a linux environment. I just checked out your commit #337 and has to change the line ending to unix line endings in my eclipse. Afterwards the error disappeared.

**Comment by pithumke** [comment]
@AlexanderBarbie Thanks for picking this up! My versions are:
```
Docker version 19.03.8, build afacb8b
docker-compose version 1.25.4, build 8d51620a
```
I did not add the "wait-for-it" script to the Docker file sharing settings, as it is already being included as a volume mount:
```
volumes:
      - ./scripts/wait-for-it:/root/scripts/wait-for-it
```
The specific console output I got was:
```
Starting cwa-server_objectstore_1 ... done
: No such file or directory
```
Upon playing around with the error and including some `pwd` and `ls` statements, I can positively say that the script exists inside the container and is being started, but still, the above error is being thrown. At this point I also suspect something along the lines of [this](https://github.com/vishnubob/wait-for-it/pull/43) or [this](https://github.com/docker/compose/issues/2301#issuecomment-320944308), but unfortunately I currently don't have the time to look into this any further.
